cowprotocol/cowswap
# [Limit orders] UI improvements

# Summary

1. Allow using ENS name as recipient
2. Disable selecting of native currency in the limit orders page (to avoid wrap/unwrap/eth-flow cases)
3. Use `feeAmount` as zero for limit order
4. Styles adjustments

# To test
1. Create a limit orders with the recipient as address and as ENS name
2. Open limit orders link with native currency symbol: `/5/limit-orders/ETH/DAI` (should redirect to `/5/limit-orders/_/DAI`)
3. Check `POST: /api/v1/orders` request, `feeAmount` should be zero
